README.md

```diff
@@ -1,24 +1,73 @@
-# README
+## Hotel Reservstion 🏩
```

Comment:
- [ ]  Kindly add a screenshot to your readme. It could be your model UML or even your API docs, or the initial screen of your app.

That way, the functionality of your app will be well illustrated, and the professionalism of the repo improved, 💪